Documentation/DownloadingTheMRTK.md

```diff
@@ -0,0 +1,36 @@
+# How to download the Mixed Reality Toolkit
+
+![](/External/ReadMeImages/MRTK_Logo_Rev.png)
+
+Recognizing there are multiple ways in which developers want to be able to get access to project through Unity, the Mixed Reality Toolkit has been engineered to be available through the following methods:
+
+## Unity Asset downloadable from the MRTK GitHub site
+
+You can download the packaged [Unity Asset from here](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases) for importing in to your project.
+
+## Unity NuGet delivery (*Coming soon*)
+
+Unity now has an asset plugin for getting access to Unity projects using the dependency delivery solution known as NuGet.
+
+Simply install the [Unity NuGet package](https://assetstore.unity.com/packages/tools/utilities/nuget-for-unity-104640) and search for the Mixed Reality Toolkit to download the project.
+
+## Unity Package Manager (*Coming soon*)
+
+As soon as it's available, the MRTK team will be working with Unity to publish the Mixed Reality Toolkit as a package through their Unity Package Manager.
+
+## Unity Asset Store (*under discussion*)
+
+It has been raised within the Mixed Reality Toolkit team that the package should be published to the Unity store.  Microsoft are in talks with Unity as to the best delivery options.  Likely this will be the Unity Package Manager.
```

Comment:
```suggestion
It has been raised within the Mixed Reality Toolkit team that the package should be published to the Unity store.  Microsoft is in talks with Unity as to the best delivery options.  Likely this will be the Unity Package Manager.
```

Reply:
I would disagree grammatically.  "Are" states they have begun conversations and are continuing.
"Is" states that are in talked with them at this moment.

maybe changing to "is currently", but I don't think it benefits from the change and still states the same as "are" :D